### PR TITLE
Document the docker settings for running in production mode

### PR DESCRIPTION
**Pull request by dhanainme**
### Document the docker settings for running in production mode #234.

Adds documentation for TS in Production env for Docker -  Shared Memory Size / User Limits for System Resources / Exposing specific ports / volumes between the host & docker env.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -161,3 +162,37 @@ torch-model-archiver --model-name densenet161 --version 1.0 --model-file /home/m
 Refer [torch-model-archiver](../model-archiver/README.md) for details.
 
 4. desnet161.mar file should be present at /home/model-server/model-store
+
+# Running TorchServe in a Production Docker Environment.
+
+You may want to consider about the following aspects / docker options when deploying torchserve in Production with Docker.
```
Comment (maaquib) [inline docker/README.md:168]:
```suggestion
You may want to consider the following aspects / docker options when deploying torchserve in Production with Docker.
```

File: docker/README.md [hunk 2]
```diff
@@ -161,3 +162,37 @@ torch-model-archiver --model-name densenet161 --version 1.0 --model-file /home/m
...
+    
+    * ```--ulimit memlock=-1``` : Maximum locked-in-memory address space. 
+    * ```--ulimit stack``` : Linux stack size 
+
+    The current ulimit values can be viewed by executing ```ulimit -a```. A more exhaustive set of options for resource constraining can be found in the Docker Documentation [here](https://docs.docker.com/config/containers/resource_constraints/), [here](https://docs.docker.com/engine/reference/commandline/run/#set-ulimits-in-container---ulimit) and [here](https://docs.docker.com/engine/reference/run/#runtime-constraints-on-resources)
+
+
+* Exposing specific ports / volumes between the host & docker env.
+
+    *  ```-p8080:p8080 -p8081:8081``` TorchServe uses 8080 / 8081 for inference & management APIs. You may want to expose these ports to the host for HTTP Requests between Docker & Host.
```
Comment (maaquib) [inline docker/README.md:186]:
```suggestion
    *  ```-p8080:p8080 -p8081:8081``` TorchServe uses default ports 8080 / 8081 for inference & management APIs. You may want to expose these ports to the host for HTTP Requests between Docker & Host.
```

File: docker/README.md [hunk 3]
```diff
@@ -161,3 +162,37 @@ torch-model-archiver --model-name densenet161 --version 1.0 --model-file /home/m
...
+    * The model store is passed to torchserve with the --model-store option. You may want want to consider using a shared volume if you prefer pre populating models in model-store directory.
```
Comment (maaquib) [inline docker/README.md:187]:
```suggestion
    * The model store is passed to torchserve with the --model-store option. You may want to consider using a shared volume if you prefer pre populating models in model-store directory.
```

**Comment by dhanainme** [comment]
Verified docker command based on instructions
```
docker run --rm --shm-size=1g \
        --ulimit memlock=-1 \
        --ulimit stack=67108864 \
        -p8080:8080 \
        -p8081:8081 \
        --mount type=bind,source=~/model_store,target=/tmp/models d3ef8f67bc41 torchserve --model-store=/tmp/models
```

**Pull request merged**